src/utils/hooks/beaconHooks.ts

```diff
@@ -1,18 +1,27 @@
 import { NetworkType } from "@airgap/beacon-wallet";
+import { get } from "lodash";
 import { useDispatch } from "react-redux";
 
 import { RawPkh } from "../../types/Address";
 import { useAppSelector } from "../redux/hooks";
-import { DAppConnectionInfo, beaconSlice } from "../redux/slices/beaconSlice";
+import { beaconSlice } from "../redux/slices/beaconSlice";
 
 /**
- * Returns connected account pkh & network by a given dAppId.
- *
- * @param dAppId - generated from dApp public key.
+ * Returns function to get connected accounts stored in {@link beaconSlice} by dAppId.
  */
-export const useGetConnectionInfo = (dAppId: string): DAppConnectionInfo | undefined => {
+export const useGetConnectedAccounts = () => {
   const beaconConnections = useAppSelector(s => s.beacon);
-  return beaconConnections[dAppId];
+
+  return (dAppId: string): RawPkh[] => Object.keys(get(beaconConnections, [dAppId], {}));
```

Comment:
`get(beaconConnections, [dAppId], {})` => `beaconConnections[dAppId] || {}`. it's a bit cleaner